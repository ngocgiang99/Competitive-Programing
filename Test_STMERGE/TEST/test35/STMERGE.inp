10
1 5
66085424 820125944 43075405 234168265 854028009 
8 1
809237756 
51960088 
688062435 
855008726 
735545821 
344712293 
468928694 
21415555 
7 9
240409678 312335036 556559418 95008296 643877471 427193030 734172448 798053160 287507168 
667318566 642556040 948006489 813247492 564198561 442909894 280910496 945788482 890754620 
935849584 245602081 821102157 717347645 586722906 686461250 327285531 262202152 353519001 
515050664 308639713 385546177 765158787 309857079 494584340 660930319 630811053 777731861 
857061225 126311766 976109034 969167526 618559445 853221660 841205841 118317712 934303628 
743258916 165399351 50987960 691465989 914682484 396714799 192362307 734645491 744507474 
477169425 996618389 198941772 340246594 321453039 259876180 582016582 707798537 148486432 
10 8
362646956 650727993 14224619 603085868 242783088 10738445 41893966 367520706 
491106778 822032819 882763313 370830669 761636272 169826944 72613754 332744001 
114694645 916115976 902678349 561441362 859142640 56881440 647575824 555029367 
419332901 450410206 328064104 775717347 210024793 858172054 981827898 87486220 
446476433 981970910 543339946 152095911 797605139 778954889 319309854 881096873 
61302626 203074735 710743354 522179399 864901189 180442274 179745252 216426742 
501603519 954355961 136303937 136583786 987510807 327194761 430766261 562762394 
976410065 997581414 287298534 566044857 775794098 267276773 542665913 537056830 
977970112 965954716 879293266 811619548 854169662 653819487 487558167 91809417 
104347059 41459523 736918444 780300154 126729442 562709406 696589560 871795236 
6 10
898496797 384633659 42140359 25364299 340767107 793615798 948483142 114867847 355971021 162868937 
542371093 516790541 293211105 17036264 530135863 593169523 272847843 215929545 741942003 95798925 
464523833 360028002 406414540 892263280 612736392 46133734 259520905 265021298 895039668 136070347 
776971822 489660712 342229219 982355654 342310912 801865061 797053306 807989451 781283973 322847040 
571281949 947525137 902809984 618629646 589316786 674365452 506082552 538137955 718613853 149998192 
917532464 681397012 651437188 85136179 213042863 851791671 398952140 104717962 656362553 98770976 
3 7
837095635 746413156 47861394 522190601 856375747 190774489 485603993 
187347418 888930956 860629139 745695446 596810898 197454581 597881064 
562891052 546655912 874163457 907122028 401275328 960695248 328230135 
10 8
999526910 650519899 8291478 799854969 194485608 710210290 510728614 172527656 
96158340 877492929 466277306 754213929 799296517 380993038 343928230 432741090 
727769825 972945500 942945919 319685111 947947085 871971186 423936411 377422670 
345105341 441947988 385128078 462860556 134613224 749199943 353941805 923470230 
811116482 27641552 513631118 918328208 239997567 494739892 515563013 283343327 
604303766 676839416 502807336 975436714 65671610 670789346 549799883 131517524 
743954401 395131893 167529220 820966274 442940224 319446776 799882791 518200602 
160761164 114135023 345282761 262537025 316968890 104045343 361592511 720923740 
996508286 120100109 284162591 991733519 535842255 413916799 135608638 795366976 
591919648 947534968 274276016 355544339 520603688 757213759 23781194 640165102 
8 10
279496225 785214838 996263788 806483336 112313995 233399439 757102419 452722900 802123805 897681503 
527492676 406134657 85045931 374218059 84554292 682391060 54915658 239187959 824467288 776076627 
34476879 745766217 619471396 230084673 248484944 197614182 777170255 629618213 938210337 953228466 
544150476 155437699 68877784 206942675 635310802 459401504 895373001 549210029 589766612 745410561 
553478794 801170534 558852232 961280363 153283823 492770915 39356643 373940196 893138445 929303113 
803668762 900531760 543100474 960537852 395633569 843866451 110505994 749886369 641571631 404203156 
540026962 2256287 204116152 577914231 199517736 835123353 197731184 383688641 841564896 716064884 
387668685 609000824 933329424 655734614 844995204 790382303 303924620 498975140 730275467 928277117 
1 5
820250901 40384277 332953085 729292119 876613271 
6 8
848510662 386739144 729779596 851365852 715484596 466197404 633431092 181873425 
26335687 679337287 898820972 18105040 662792932 956943424 30686007 346415791 
564751663 244799546 819567068 390489730 703803151 900187435 738456030 708272735 
662060326 287162696 939614724 587087513 313769051 802400518 615983954 663286216 
7735251 243639578 989183245 472514750 971751667 582294137 918583483 40668307 
806803295 535570008 78277663 75370137 544987901 900580907 266124096 221460071 
