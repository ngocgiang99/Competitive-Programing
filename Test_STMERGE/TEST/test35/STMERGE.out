66085424
21415555
287507168
104347059
127276538
485603993
172527656
387668685
373337362
161383051
